.dccache

```diff
@@ -0,0 +1 @@
torProfilePage/contributor-profile-page.tsx":[6450,1674743190259.6145,"4199156df5920216137726e46993402da533ab7c03959ac25056159b6711188f"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/components/atoms/Avatar/avatar-hover-card.tsx":[1235,1674743308249.0305,"00036e378aa9d78cf1fa5d7cd45422e73f78febc9097938ac641f0840895812f"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/components/molecules/InsightHeader/insight-header.tsx":[2197,1674743190255.6145,"4649d6608463747b1a4b5101962884b2d0bb9adb8de3b720fa6242f58ef4f9af"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/stories/molecules/contributor-table.stories.tsx":[1338,1674743190283.6147,"6a8b0c5fefcec1e442e93d4c7e1b65bea4c14ec536a722e10024090e1bef5cab"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/components/molecules/CardProfile/card-profile.tsx":[1606,1674743190255.6145,"e0f61b0e65048bc0ccfbb2a9206fe35f1a4c4972deb96e0d6ddcf6f012c294e2"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/components/atoms/InsightBadge/insight-badge.tsx":[422,1674743190251.6145,"8fe6959ad9deb2b62fdd83886c93e4440d4128497eddb4393d3ea94920fd5f15"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/components/atoms/LanguagePill/LanguagePill.tsx":[1271,1674743190251.6145,"54a16fdea966cb7bf5716a07ee73bbb278554c32449611b36a4e35edd02c6daa"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/components/molecules/PullRequestSocialCard/pull-request-social-card.tsx":[2773,1674743190259.6145,"00f643e80eb63d3e1f5ae8eeeb2c2abd6a7034bc5c738e96d4d3e8abee6cb393"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/components/molecules/PullRequestTable/pull-request-table.tsx":[2223,1674743190259.6145,"3acef3388be73a716f04de125eee87a63fb6effc330a7743e03fcf1cebcfdae2"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/lib/hooks/useFetchUser.ts":[471,1674743190271.6147,"07868fa2a1109834c9d4f084fbb1219a9d4134a4692e44ca3f6b38b5558a6762"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/stories/atoms/insight-badge.stories.tsx":[459,1674743190283.6147,"0d40ea91a0041ba44f5c69802eabb4997d58f6e1ba5a8953c51421b941a42da2"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/stories/atoms/language-pill.stories.tsx":[614,1674743190283.6147,"b00134689fde7040722873cf18317897cfb1ab9d87d422d2b049541885e5cc0c"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/stories/molecules/pull-request-social-card.stories.tsx":[1087,1674743190283.6147,"9ff3212e55a47cde3dd8ae90c81f2bd60e13dce5053f9366ad2bed4a59761266"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/stories/organisms/user-settings-page.stories.tsx":[408,1674743190283.6147,"044de26aac0f81afea419335c92df49d9d35d3ced082c5beb7dec24368857745"],"/home/takanome_dev/Projects/OpenSource/open-sauced/insights/components/organisms/UserSettingsPage/user-settings-page.tsx":[4914,1674743190263.6145,"354d2178990c897487a58f2d5212c0312623037d27c6533ebe04aa3285438505"]}
```

Comment:
Delete this?

Reply:
Yeah, I thought I removed it. Thanks :+1: